src/App.js

```diff
@@ -1,5 +1,50 @@
+const BridgeGame = require('./BridgeGame');
+const InputView = require('./InputView');
+const OutputView = require('./OutputView');
+const { COMMAND } = require('./utils/constructor');
+const { generate } = require('./BridgeRandomNumberGenerator');
+
 class App {
-  play() {}
+  #game
+
+  constructor() {
+    this.#game = new BridgeGame();
+  }
+
+  initBridge(size) {
+    this.#game.setBridge(size, generate);
+    InputView.readMoving((movement) => this.runGame(movement));
+  }
+
+  runGame(movement) {
+    const result = this.#game.move(movement);
```

Comment:
이렇게 구조 분해 할당을 사용할 수 있답니다!
```suggestion
    const { isGameEnd, isCorrect } = this.#game.move(movement);
```